### PR TITLE
feat: add vovo tvl

### PR DESCRIPTION
**Pull request by flashdebugger**
##### Twitter Link:
https://twitter.com/VovoFinance

##### List of audit links if any:
https://github.com/VovoFinance/VovoProducts/tree/main/audits

##### Website Link:
https://vovo.finance/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
logo-bg
logo-sm

##### Current TVL:
580.08 k 

##### Chain:
Arbitrum

##### Short Description (to be shown on DefiLlama):
Structured products with various risk-returns and payoff structures through packaged financial instruments.

##### Token address and ticker if any:
No

##### forkedFrom (Does your project originate from another project):
No

### CODE REVIEW

File: projects/vovo/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  const totalTvl =
+    (await usdcVaultsTVL(block)) + (await glpVaultsTVL(block));
+
+  return {
+    'usd-coin': totalTvl,
+  };
+}
+
+module.exports = {
+  arbitrum: {
```
Comment (waynebruce0x) [inline projects/vovo/index.js:71]: can you add 'misrepresentedTokens: true' to the exports since we're converting to USDC

Comment (flashdebugger) [reply projects/vovo/index.js:71]: sure

**Comment by llamatester** [comment]
The adapter at projects/vovo exports TVL: 
        
 
 
    arbitrum                  577.48 k
    
    total                    577.48 k

**Comment by llamatester** [comment]
The adapter at projects/vovo exports TVL: 
        
 
 
    arbitrum                  577.49 k
    
    total                    577.49 k

**Pull request merged**